### PR TITLE
Add oVirt OWNERS

### PR DESCRIPTION
**Pull request by gekorob**

### CODE REVIEW

File: data/data/ovirt/OWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+# This file just uses aliases defined in OWNERS_ALIASES.
+
+approvers:
```
Comment (abhinavdahiya) [inline data/data/ovirt/OWNERS:4]: make sure all the OWNERS files look like https://github.com/openshift/installer/blob/master/data/data/aws/OWNERS

both approvers and reviewers should exist...

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -56,3 +56,9 @@ aliases:
     - fabianofranz
     - jhixson74
   azure-reviewers:
+  ovirt-approvers:
+    - rgolangh
+    - Gal-Zaidman
+    - dougsland
+    - gekorob
+  ovirt-reviewers:
```
Comment (abhinavdahiya) [inline OWNERS_ALIASES:64]: let's keep `rgolangh` as approvers list, everybody else should be in reviewers list.

- make sure there are atleast 2 people in reviewers.
- duplicates in approvers and reviewers is okay.

Comment (gekorob) [reply OWNERS_ALIASES]: done

File: data/data/ovirt/OWNERS (hunk 1 above)
Comment (gekorob) [reply data/data/ovirt/OWNERS:4]: Got it, done

File: OWNERS_ALIASES (hunk 2 above)
Comment (sdodson) [reply OWNERS_ALIASES]: I think we were expecting `rgolangh` to be in both `ovirt-approvers` and `ovirt-reviewers` ?

Comment (gekorob) [reply OWNERS_ALIASES]: no pb, done

**Comment by gekorob** [comment]
@abhinavdahiya @sdodson do we need a particular BZ to merge this owners PR?

**Comment by sdodson** [comment]
> @abhinavdahiya @sdodson do we need a particular BZ to merge this owners PR?

No, I'll override.

**Pull request merged**